### ISSUE TITLE
Staging Review finding: Page title does not update on navigation

### ISSUE THREAD

**Issue by jasonday** [department-of-veterans-affairs/va.gov-team#119433]
_Need help? Please review [how to read a Staging Review ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/anatomy-of-a-staging-review-issue-ticket). Tag `@platform-governance-team-members` on Slack if you need further assistance._

## Product Information
**Team:** Iterate, Innovate, & Run (IIR)
**Product:** Paperless Delivery
**Feature:** Paperless Delivery

## Findings details
**VA.gov Experience Standard - issue:** [User experience is not consistent with the VA.gov design principles.](https://design.va.gov/about/experience-standards/#users-experience-is-consistent-with-the-vagov-design-principles)
**VA.gov Experience Standard - category:** [Consistency](https://design.va.gov/about/experience-standards/#category-consistency)
**Launch-blocking:** No
**Design System review:** No
**Collab Cycle Reviewer:** @jasonday (Accessibility), @erinrwhite (IA)

### Description

On Update your contact email address screen, the page title is not updated using the previous screen's page title

## Recommended action

Update the page title to reference the H1 on this screen.

IA recommendation - update to `Update your contact email | Veterans Affairs` based on most recent guidance for title tags in the content style guide.

## References

- **Accessibility Defect Severity:** [4: Low. Consider fixing or exploring in 2-4 sprints post-launch.](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-defect-severity-rubric#Accessibilitydefectseverityrubric-a11y-defect-4)
- **WCAG Success Criteria:** [WCAG 2.4.2 Page Titled - Level A](https://www.w3.org/TR/WCAG22/#page-titled)
- **Modality:** Screen reader
- **Design System Component:** N/A
- **Design System Pattern or Template:** N/A
- **Design System Foundation:** N/A
- **Content Style Guide:** [Title tags](https://design.va.gov/content-style-guide/title-tags)
- **Context:** N/A

---

### Next Steps for the VFS Team

_How did this touchpoint go? [Give feedback on the Collaboration Cycle](https://dj540s05.optimalworkshop.com/questions/f54355271969f160ef41182ba9ceb61c) at any time._
- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members` with any questions or to get help validating the issue.
- [ ] **Ticket ownership:** If you believe that this issue is out of scope, not your team's responsibility, or a Design System issue, comment and tag `@platform-governance-team-members` on your team channel in Slack to provide an explanation and who you believe is responsible. The Governance team will follow up.
- [ ] **Close this ticket** when the issue has been resolved or validated by your Product Owner.
